<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tankWindow</class>
 <widget class="QMainWindow" name="tankWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>人工智能履带坦克</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>720</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabMenuWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>660</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>660</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>660</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>1021</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#2F4056;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <widget class="QWidget" name="tab_control">
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>运动控制</string>
     </attribute>
     <widget class="QCheckBox" name="avoid">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开启</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>70</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>循迹模式</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_backward">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>290</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(99, 180, 255);color:rgb(243, 243, 243);</string>
      </property>
      <property name="text">
       <string>后退</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="keyboard_control">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开启</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="travel">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开启</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_rotate">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>340</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 127);color:rgb(66, 66, 66);</string>
      </property>
      <property name="text">
       <string>旋转</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_right">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>240</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(99, 180, 255);color:rgb(243, 243, 243);</string>
      </property>
      <property name="text">
       <string>向左</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>150</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(85, 170, 255)</string>
      </property>
      <property name="text">
       <string>行动控制</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_cam_left">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>490</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#01AAED;color:#F3F3F3;</string>
      </property>
      <property name="text">
       <string>向右</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_forward">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>190</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string extracomment="go_front"/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(99, 180, 255);color:rgb(243, 243, 243);</string>
      </property>
      <property name="text">
       <string>前进</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_cam_down">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>440</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#01AAED;color:#F3F3F3;</string>
      </property>
      <property name="text">
       <string>向上</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>避障模式</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_stop">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>340</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 85, 0);color:rgb(243, 243, 243);</string>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_cam_up">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>540</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#01AAED;color:#F3F3F3;</string>
      </property>
      <property name="text">
       <string>向下</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>110</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>键盘控制</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>400</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(85, 170, 255)</string>
      </property>
      <property name="text">
       <string>摄像头控制</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_cam_right">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>490</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#01AAED;color:#F3F3F3;</string>
      </property>
      <property name="text">
       <string>向左</string>
      </property>
     </widget>
     <widget class="QPushButton" name="car_left">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>240</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(99, 180, 255);color:rgb(243, 243, 243);</string>
      </property>
      <property name="text">
       <string>向右</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>540</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>摄像头画面</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="camera_switch">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>550</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开启</string>
      </property>
     </widget>
     <widget class="QLabel" name="origin_camera">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#F2F2F2</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_voice">
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255,255,255,0.8);</string>
     </property>
     <attribute name="title">
      <string>语音技术</string>
     </attribute>
     <widget class="QPushButton" name="voice_control">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>80</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>语音控制</string>
      </property>
     </widget>
     <widget class="QPushButton" name="voice_text_composite">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>150</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>语音合成</string>
      </property>
     </widget>
     <widget class="QPushButton" name="voice_text_chat">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>480</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>文字对话</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>501</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>语音对话：唤醒词为佩奇佩奇，唤醒成功后听到提示音进行语音对话。</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_composite_text">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>591</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>   语音合成会将输入的文字合成语音并播放，请输入文字...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="voice_chat">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>40</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>语音对话</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="voice_chat_list">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>711</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align:center;font-size:15px;padding-left:150px;line-height:40px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_send_text">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>480</y>
        <width>631</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>   请输入聊天文字内容...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>651</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>语音控制：唤醒词为佩奇佩奇，唤醒成功后说前进、后退、左转、右转、旋转等进行控制。</string>
      </property>
     </widget>
     <widget class="QPushButton" name="voice_audio_start">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>550</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>录音</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>550</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>录音播放：使用车载麦克阵列及音箱进行录音及播放。</string>
      </property>
     </widget>
     <widget class="QPushButton" name="voice_audio_stop">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>550</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>停止录音</string>
      </property>
     </widget>
     <widget class="QPushButton" name="voice_audio_play">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>550</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>播放</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_video">
     <attribute name="title">
      <string>视频处理</string>
     </attribute>
     <widget class="QPushButton" name="camera_origin">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>原始画面</string>
      </property>
     </widget>
     <widget class="QPushButton" name="camera_face">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>250</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>人脸识别</string>
      </property>
     </widget>
     <widget class="QPushButton" name="camera_eyes">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>320</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>眼部识别</string>
      </property>
     </widget>
     <widget class="QPushButton" name="camera_close">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>390</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QLabel" name="video_pannel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">backgroud-color:#F2F2F2</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_code">
     <attribute name="title">
      <string>代码编辑</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#01AAED;color:#F1F1F1;width:80px;</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="styleSheet">
     <string notr="true">width:95px;border:2px solid #F1F1F1;</string>
    </property>
    <property name="title">
     <string>设备</string>
    </property>
    <addaction name="actionChange"/>
    <addaction name="actionExit"/>
    <addaction name="actionlogin"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="styleSheet">
     <string notr="true">width:95px;border:2px solid #F1F1F1;</string>
    </property>
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>切换</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="actionsds">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="actionlogin">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
